src/FSharpLint.Console/paket.references

```diff
@@ -1,4 +1,5 @@
 Argu
 FSharp.Compiler.Service
 FSharp.Core
-Microsoft.SourceLink.GitHub
\ No newline at end of file
+Microsoft.SourceLink.GitHub
+Sarif.Sdk import_targets: false
```

Comment:
This ```import_targets: false``` is because I was initially getting this error from builds after adding the nuget package reference:
```
Exec: EXEC(0,0): error : Unable to load one or more of the requested types. Retrieve the LoaderExceptions property for more information.
Exec: C:\Users\Richard Webb\.nuget\packages\microsoft.diagnostics.tracing.eventregister\1.1.28\build\Microsoft.Diagnostics.Tracing.EventRegister.targets(132,5): error MSB3073: The command ""C:\Users\Richard Webb\.nuget\packages\microsoft.diagnostics.tracing.eventregister\1.1.28\build\eventRegister.exe" -DumpRegDlls @"S:\GitHubForks\FSharpLint\src\FSharpLint.Console\bin\Release\net6.0\dotnet-fsharplint.eventRegister.rsp" "S:\GitHubForks\FSharpLint\src\FSharpLint.Console\bin\Release\net6.0\dotnet-fsharplint.dll" " exited with code 1.
```
And I'm not sure why (I haven't used any of that eventregister stuff myself for years, and don't recall the defaults of it)

Reply:
please add this info in the commit msg